### PR TITLE
Invoke example

### PR DESCRIPTION
**Pull request by OR13**
We need a simple HTTP Signature Invoke example, with keys, and all parameters.

### CODE REVIEW

**Comment by OR13** [comment]
currently seeing strange failures regarding invoke http signature implementation.

**Review by msporny — APPROVED** [review]

File: packages/data-vault-client/src/authorization/zcap.sanity.test.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { signCapabilityInvocation } from 'http-signature-zcap-invoke';
+import { CapabilityInvoker } from '../CapabilityInvoker';
+import * as ed25519 from '@transmute/did-key-ed25519';
+
+import base64url from 'base64url';
```
Comment (dlongley) [inline packages/data-vault-client/src/authorization/zcap.sanity.test.ts:5]: Should this be:

```suggestion
import base64url from 'base64url-universal';
```

To match the package.json file? If so, another line below should change...

**Comment by OR13** [comment]
I am going to try and tackle this on the existing edv client first... see https://github.com/digitalbazaar/edv-client/pull/76

**Comment by OR13** [comment]
I'm going to close this, and try again when edv-client has commited test vectors.

**Pull request closed without merging**